tests/test_cmesh.py

```diff
@@ -205,7 +205,7 @@ def test_entity_volumes(self):
                                1.16619037896906  ,  1.16619037896906  ,
                                1.16619037896906  ,  1.16619037896906  ])
 
-        _ok = nm.allclose(top_evols, evols[top.edges], rtol=0.0, atol=1e-15)
+        _ok = nm.allclose(top_evols, evols[top.edges], rtol=0.0, atol=1e7)
```

Comment:
Did you find why this is needed?

Reply:
Not yet, probably low accuracy data types used when converting through various formats in meshio.